### PR TITLE
Push Message Changes

### PR DESCRIPTION
**Pull request by kdvlr**
- Made Push message generic
- Updated existing scripts to use new push message format
- Added push message for start of dashcam and music copy.

### CODE REVIEW

File: run/send-push-message [hunk 1]
```diff
@@ -1,8 +1,7 @@
 #!/bin/bash -eu
 
-NUM_FILES_MOVED="$1"
-TITLE=${2:-"Dashcam Copy Complete"}
-MESSAGE="$NUM_FILES_MOVED file(s) were copied."
+TITLE="$1"
```
Comment (cjwang18) [inline run/send-push-message:3]: Since all of the push messages start off with "TeslaUSB:", would it make sense to just prepend it here?

`TITLE="TeslaUSB: $1"`

`TITLE` would be the main action and `MESSAGE` can be the optional details (how many files were moved, skipped, failed).

Comment (kdvlr) [reply run/send-push-message:3]: Valid Point. I have other things on the Pi (and elsewhere) that send a push notification using this utility, so I have used TeslaUSB: prefix to tell me which component is sending me the push notification. 

Ideally the push message should not be too wordy, so I have shortened the title to be only ```TeslaUSB:``` always so it makes even more sense to have it be a constant

Comment (kdvlr) [reply run/send-push-message:3]: Thinking it through, I think that Title should uniquely identify the car if someone has more than 1 tesla so they know where the notification is coming from.

File: run/archiveloop [hunk 2]
```diff
@@ -246,18 +246,22 @@ function disconnect_usb_drives_from_host () {
...
+  
   /root/bin/archive-clips.sh
 
   unmount_cam_file
 }
 
 function copy_music_files () {
   log "Starting music sync..."
+  
+  /root/bin/send-push-message "TeslaUSB: Music copy started" "Music copy started at $(date)"
```
Comment (marcone) [inline run/archiveloop]: I could see this getting annoying pretty quickly, since it will be sent even if it ends up not copying anything because there was nothing new to copy.

**Review by marcone — COMMENTED** [review]
Could you squash this into one commit? I think it'll be easier to review that way because right now there are changes upon changes.

**Comment by kdvlr** [comment]
CLosing this because my git foo failed me.
